### PR TITLE
Translate Premium Theme string to Lithuanian

### PR DESCRIPTION
**Pull request by mistermantas**
Just this one single string. Hopefully I did not mess up anything since I am doing this on my phone! 👍

### CODE REVIEW

File: app/src/main/res/values-lt/strings.xml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <resources>
+    `<string name="premium_theme">Aukšt. kokyb. tema</string>`
```
Comment (k0shk0sh) [inline app/src/main/res/values-lt/strings.xml]: yet you did :p 

can you remove the single quote ? :D

**Comment by mistermantas** [comment]
Checks failed? Oh boy.

File: app/src/main/res/values-lt/strings.xml (hunk 1 above)
Comment (mistermantas) [inline app/src/main/res/values-lt/strings.xml]: How did I even add that? OK LOL.

**Comment by k0shk0sh** [comment]
ignore travis :D .. i have problem with it, but please remove the single quotes

**Comment by mistermantas** [comment]
Should be good now. I’d squash those commits but am currently on the web UI so I have no idea how to do that without getting to my CLI on my PC.

**Comment by k0shk0sh** [comment]
its alright :) no worries about it.

**Comment by yakov116** [comment]
@arkon Test PR 2

**Pull request merged**